scripts/issue/34/issue34.py

```diff
@@ -21,27 +21,43 @@ def calc_deposit_amount(amount):
     return POOL.get_dy(0, 1, amount)
 
 def main():
-    
-    safe = GreatApeSafe(registry.eth.badger_wallets.techops_multisig)
+    safe = GreatApeSafe(registry.eth.badger_wallets.treasury_ops_multisig)
+    safe.init_curve()
     safe.init_curve_v2()
-
-    if 'fork' in network.show_active():
-        WBTC = MintableForkToken(registry.eth.treasury_tokens.WBTC)
-        BADGER = MintableForkToken(registry.eth.treasury_tokens.BADGER)
-        WBTC._mint_for_testing(safe.address, 3.5 * 10**WBTC.decimals())
-        BADGER._mint_for_testing(safe.address, 15000 * 10**BADGER.decimals())
-    else:
-        WBTC = interface.ERC20(registry.eth.treasury_tokens.WBTC)
-        BADGER = interface.ERC20(registry.eth.treasury_tokens.BADGER)
+    
+    WBTC = interface.ERC20(registry.eth.treasury_tokens.WBTC)
+    BADGER = interface.ERC20(registry.eth.treasury_tokens.BADGER)
+    
+    bcrvIbBTC = interface.ISettV4h(
+        registry.eth.sett_vaults.bcrvIbBTC, owner=safe.address
+    )
+    
+    crvIbBTC = safe.contract(registry.eth.treasury_tokens.crvIbBTC)
     
     safe.take_snapshot(tokens=[BADGER, WBTC])
     
+    # get amounts to deposit into factory pool
     badger_amount = AMOUNT_BADGER_ETH * 10**BADGER.decimals()
     wbtc_amount = calc_deposit_amount(badger_amount)
```

Comment:
get_dy cannot be used for finding out what the wbtc_amount is. the pool is currently not deep enough, and returns 23560388.

my suggestion here would be to do it manually and divide by price_oracle (and not price_scale). this way we add liquidity in a ratio that is truthful to the ratio we want to pool to be at.